tox.ini

```diff
@@ -2,29 +2,30 @@
 skipsdist=True
 envlist=
     lint,
-    py{36}-dj{22}-wag{27,latest},
-    py{39}-dj{22,32,40}-wag{latest}
+    python{3.7,3.8,3.9,3.10}-django{3.1,3.2,4.0}-wagtail{2.15,2.16,3.0}
```

Comment:
As with the GitHub Actions changes, while these might make sense in isolation, they're inconsistent with the other Python/Django/Wagtail libraries we maintain. I'd like to see wholesale changes to our test environments proposed in a separate PR.

Reply:
I made changes here too along the lines of: https://github.com/cfpb/wagtail-flags/blob/main/tox.ini